CoastalResilienceNetwork/GeositeFramework
# Make chosen and jquery-ui available globally

### Overview

This PR makes the chosen and jquery-ui libraries available to plugins by changing them into use module imports rather than cdn-loaded scripts, them importing them into the BasePlugin.js file after which they'll be available to plugins inheriting from it. To accommodate loading them via `use`, this supplants using script tags with importing the minified jquery-ui and chosen files.

This enables both libraries to be used without any additional imports.

To demonstrate using these, I also added a range slider, accordion menu, and multiselect box to the Identify Point plugin -- mostly by just copying the code from the style guide: http://coastalresiliencenetwork.github.io/GeositeFramework/snippet_index.html

Connects #903 

### Screenshot

<img width="401" alt="screen shot 2017-03-08 at 4 01 56 pm" src="https://cloud.githubusercontent.com/assets/4165523/23723825/97e275ba-0418-11e7-9497-75958d87a7cf.png">

### Notes

~~Still two todos here, one which is to make the "pips" library available to the slider to show interval ticks as in the "Slider" example here: http://coastalresiliencenetwork.github.io/GeositeFramework/snippet_index.html~~

~~The other's to update the wiki with example code snippets.~~

Pips is now available via some updates to the imported file; the examples used in the sample plugin were pulled directly from the style guide, and the Wiki already mentions using snippets from the styleguide here:

https://github.com/CoastalResilienceNetwork/GeositeFramework/wiki/Styleguide

### Testing

 * rebuild this branch and bring it up in the browser
 * open identify point and verify that you see the new example elements and that they function as they should; compare them with what's in the style guide linked above
 * verify that you don't see any errors in the console
